tests/testthat/test-import_file.R

```diff
@@ -0,0 +1,9 @@
+test_that("file import runs successfully", {
+  vx <- import_file("dbtest/VACCINES.csv")
```

Comment:
directory is "tests/testthat/dbtest/VACCINES.csv"

Reply:
This file does not get run from the root, but from the `tests/testthat/` directory; the path therefore is correct!